src/components/molecules/headers/Header.tsx

```diff
@@ -1,30 +1,7 @@
-import React from 'react'
-import Box from '@mui/material/Box'
-import { MainHeading } from '../../atoms/mainHeading/MainHeading'
-import { SubHeading } from '../../atoms/subHeading/SubHeading'
+import { NavBar } from '../../templates/NavBar'
 
 export function Header() {
   return (
-    <Box
-      sx={{
-        margin: '20px',
-        fontWeight: 'bold',
-        display: 'flex',
-        justifyContent: 'space-between',
-        alignItems: 'center',
-        color: 'white',
-      }}
-    >
-      <Box
-        sx={{
-          display: 'flex',
-          flexDirection: 'column',
-          alignItems: 'flex-start',
-        }}
-      >
-        <MainHeading mainHeading="KURT" />
-        <SubHeading subHeading="SENSENBRENNER" />
-      </Box>
-    </Box>
+    <NavBar />
```

Comment:
Can we just use the navbar in the `app` component so that we don't have this otherwise empty component?

Reply:
100%!
No reason for this component anymore since everything we need is in the NavBar component.